onnxruntime/core/optimizer/graph_transformer_utils.cc

```diff
@@ -212,7 +213,7 @@ InlinedVector<std::unique_ptr<GraphTransformer>> GenerateTransformers(
       transformers.emplace_back(std::make_unique<ConstantSharing>(no_limit_empty_ep_list, excluded_initializers));
 
       transformers.emplace_back(std::make_unique<CommonSubexpressionElimination>());
-      transformers.emplace_back(std::make_unique<ConstantFolding>(cpu_execution_provider, !disable_quant_qdq));
+      transformers.emplace_back(std::make_unique<ConstantFolding>(cpu_execution_provider, !disable_quant_qdq, allocators));
```

Comment:
>allocators [](http://example.com/codeflow?start=110&length=10)

the ConstantFolding optimizer only support CPU EP, so it is arguable that whether we need to have allocator map, or just use a temporary CPU allocator